CONTRIBUTING.md

```diff
@@ -0,0 +1,159 @@
+
+#### Table Of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+
+2. [Help! I don't want to read this whole thing, I just have one question. :mag_right:](#help!-i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+3. [How Can I Contribute?](#how-can-i-contribute)
+    1. [Reporting Bugs](#reporting-bugs)
+    1. [Suggesting Enhancements](#suggesting-enhancements)
+    1. [Pull Requests](#pull-requests)
+
+4. [Styleguides](#styleguides)
+    1. [Git Commit Messages](#git-commit-messages)
+    1. [JavaScript Styleguide](#javascript-styleguide)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Lisk-JS Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [info@lisk.io](mailto:info@lisk.io).
+
+## Project License
+
+Every repository within LiskHQ comes with a LICENSE file. Please read it carefully before commiting your code to one of the repositories.
+
+
+## Help! I don't want to read this whole thing, I just have a question. :mag_right:
+
+Lisk is an open-source decentralized project, there are many ways and platforms to get help. These are some of them:
+
+* [Discuss and Ask on Reddit](https://www.reddit.com/r/Lisk/)
+* [Lisk FAQ](https://docs.lisk.io/docs/faq)
+
+If you prefer to chat with LiskHQ and other developers directly:
+
+* [Join the LiskHQ Gitter](https://gitter.im/LiskHQ/lisk)
+* Even though Gitter is a chat service, sometimes it takes several hours for community members to respond &mdash; please be patient!
+
+## How Can I Contribute?
+
+LiskHQ uses GitHub as its sole source of truth. Everything happens here. Lightcurve employees who contribute to Lisk are expected to do so in the same way as everyone else. In other words, this document applies equally to all contributors.
+
+### `master` is unsafe for production use
+
+We will do our best to keep `master` in good shape, with tests passing at all times. But in order to move fast, we will make API changes that your application might not be compatible with. We will do our best to communicate these changes and always version appropriately so you can lock into a specific, stable version.
+
+### Pull Requests
+
+In case you've never submitted a pull request (PR) via GitHub before, please read [this short tutorial](https://help.github.com/articles/creating-a-pull-request). If you've submitted a PR before, there should be nothing surprising about our procedures for Lisk.
+
+*Before* submitting a pull request, please make sure the following is done:
+
+1. Fork the repo and create your branch from `master`.
+2. Add tests to the code you have contributed!
+All new code must come with complete test coverage.
+End all files with a newline. Generally your code should conform to the .editorconfig rules.
```

Comment:
"In general, your code should conform to the rules listed in the `.editorconfig` file. There are [plugins for most editors/IDEs](http://editorconfig.org/#download) to do this for you automatically."